grpcclient/lc_api.go

```diff
@@ -70,6 +73,26 @@ func (c *LcClient) SetMulti(ctx context.Context, req *schema.SetMultiRequest) (*
 	return c.ServiceClient.SetMulti(ctx, req)
 }
 
+// VCNSetArtifacts ...
+func (c *LcClient) VCNSetArtifacts(ctx context.Context, req *schema.VCNArtifactsRequest) (*schema.VCNArtifactsResponse, error) {
+	if c.PrivateKey != nil {
```

Comment:
Please move the calculation of the signature to VCN

Reply:
Moved.